plugins/org.wso2.developerstudio.eclipse.esb.project/src/org/wso2/developerstudio/eclipse/esb/project/servlets/FunctionServerConstants.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2009-2018 WSO2, Inc. (http://wso2.com)
```

Comment:
Why, above copyright year says 2009-2018?

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/1063